src/validators/callout-component-input-address.validator.ts

```diff
@@ -8,7 +8,7 @@ export const calloutComponentInputAddressValidator: ValidatorCalloutComponent<
   CalloutComponentInputAddressSchema
 > = (
   _schema: CalloutComponentInputAddressSchema,
-  _answer: CalloutResponseAnswer,
+  _answer: CalloutResponseAnswer | undefined,
```

Comment:
I don't think you need `| undefined` here? My idea was that these validators always expect an answer, so you don't need to do the if required check here.

The `calloutComponentInputValidator` handles the `undefined` case so all the other validators can just work with a defined answer.